udata_front/theme/gouvfr/assets/js/components/dataset/QualityScore/QualityScoreTooltip/QualityScoreTooltip.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <p v-bind="$attrs" :aria-describedby="id" ref="contentRef">
+    <QualityScore :score="dataset.quality.score" />
+  </p>
+  <div class="fr-tooltip fr-placement" :id="id" role="tooltip" aria-hidden="true">
+    <QualityScoreTooltipContent :dataset="dataset" />
```

Comment:
The `QualityScoreTooltipContent` is (and will always be?) only used here? Why not inline the logic since the two components are really simple? And why not use the `Tooltip` component here to avoid managing the random ID?

Reply:
I don't recall why I made this one. It seems that Tooltip can be used here too.

Regarding `QualityScoreTooltipContent`, now that the other QualityScore components are in `@datagouv/components`, it should be moved there too to remove duplicates but I think it can be done in a future PR.